echemdb/database.py

```diff
@@ -1,5 +1,5 @@
 r"""
-A Database of Cyclic Voltammograms.
+A Database of datapackages with units.
```

Comment:
```suggestion
A collection of datapackages with units.
```

It's always best not to repeat the information that is already given by the name of the object we're describing.